packages/kbn-es-archiver/src/lib/indices/kibana_index.ts

```diff
@@ -75,9 +78,9 @@ function isKibanaIndex(index?: string): index is string {
   );
```

Comment:
I didn't change this code so I can't select the whole block, but it looks like the `isKibanaIndex` function doesn't match up with the TS doc comment.

The comment says "(e.g. we don't want to remove .kibana_task_manager or the like)", but it looks like the check was later modified to match the `.kibana_task_manager` index anyway.

At any rate, the change I made modifies the index pattern that is fetched, and `isKibanaIndex` is only used later to filter the results, so this disparity doesn't really matter in practice. But I thought I'd point it out.